src/AppLovinMAXAdView.js

```diff
@@ -128,6 +141,8 @@ const AdView = (props) => {
   return (
     <AppLovinMAXAdView
       style={{...style, ...dimensions}}
+      extraParameter={checkExtraParameter('extraParameter', extraParameter)}
+      localExtraParameter={checkExtraParameter('localExtraParameter', localExtraParameter)}
```

Comment:
since we're injecting a dictionary which ca contain multipel values:
```suggestion
      extraParameters={checkExtraParameter('extraParameter', extraParameter)}
      localExtraParameters={checkExtraParameter('localExtraParameter', localExtraParameter)}
```

Reply:
Updated.